### DISCUSSION TITLE
Why must I include the :10000 for custom domain?

### DISCUSSION THREAD

**Discussion by hendrikjap** [Budibase/budibase#3436] [Help] [answered]
![image](https://user-images.githubusercontent.com/55825618/142407307-7b83fb34-4248-4bc0-a834-958f04b22369.png)
Hi guys, I'm self hosting in the digital ocean and I find that I must include customdomain.com:10000 in order to access the portal. Why is it so?

**Comment by shogunpurple** [comment, marked as answer]
@ely2727 it is because budibase is exposed on port 10000 by default. In order to remove the 10000 in your custom domain, you must expose budibase at port 80 or 443 on your server, or run a reverse proxy that can point to the local budibase running on 10000.

you can find a guide to do that here:

https://docs.budibase.com/self-hosting/hosting-settings/advanced-options/reverse-proxy#simple-https-setup-with-caddy

**Reply by hendrikjap** [reply]
@shogunpurple, thanks for your reply.
